src/server/serverEntry.tsx

```diff
@@ -121,8 +121,11 @@ router.post(
   handleAdyen3dsPostRedirect,
 )
 
-serverSideRoutes.forEach((route) => {
-  router.get(route.path, getPage(route))
+routes.forEach((route) => {
```

Comment:
Why don't we filter out the non-production routes here as well?

Reply:
I'm not sure I understand what you mean? The non-production routes are still needed, or? Do you mean the client-side routes?

Reply:
Hmm... I guess I was thinking of mimicking the same setup as for client side routes, i.e. filter out routes that match: `isProductionEnvironment && isHiddenInProd`

Reply:
Ah, OK, I see. Maybe we can skip that for now though?

Reply:
So I actually tried to do this but didn't really get anywhere since Koa isn't behaving at all like I expect it to, and since we're moving away from this setup anyway I don't think it's worth spending more time on that :)